src/promptflow-core/promptflow/executor/_tool_resolver.py

```diff
@@ -21,6 +20,7 @@
 from promptflow.contracts.flow import InputAssignment, InputValueType, Node, ToolSource, ToolSourceType
 from promptflow.contracts.tool import ConnectionType, Tool, ToolType, ValueType
 from promptflow.contracts.types import AssistantDefinition, PromptTemplate
+from promptflow.core._connection_provider._dict_connection_provider import ConnectionProvider
```

Comment:
import path should be `from promptflow.connections import ConnectionProvider`

Reply:
Refined.